source/engine/engine/ComponentFactories.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "ncengine/NcFwd.h"
+
+namespace nc
+{
+auto CreateAudioSource(Entity entity, void*) -> audio::AudioSource;
```

Comment:
missed unit test for these

Reply:
As discussed, these are a bit more involved than what we want in our unit tests. Time is better spent removing the problematic bits. Issue added.